web/src/elements/ak-locale-context/ak-locale-context.ts

```diff
@@ -2,13 +2,11 @@ import { EVENT_LOCALE_CHANGE } from "@goauthentik/common/constants";
 import { EVENT_LOCALE_REQUEST } from "@goauthentik/common/constants";
 import { customEvent, isCustomEvent } from "@goauthentik/elements/utils/customEvents";
 
```

Comment:
It turns out, lit-locale does its own context management. We don't need to provide our own context at all in this space, and that's one less listener we need to attach to the DOM.